modules/nf-core/mosdepth/main.nf

```diff
@@ -8,9 +8,9 @@ process MOSDEPTH {
         'quay.io/biocontainers/mosdepth:0.3.3--hdfd78af_1'}"
 
     input:
-    tuple val(meta), path(bam), path(bai)
-    path  bed
-    path  fasta
+    tuple val(meta),  path(bam), path(bai)
```

Comment:
there is an extra space there, cam fix it upstream though, not important, just to note for later